fastai/tabular/core.py

```diff
@@ -238,11 +238,11 @@ def _decode_cats(voc, c): return c.map(dict(enumerate(voc[c.name].items)))
 class Categorify(TabularProc):
     "Transform the categorical variables to something similar to `pd.Categorical`"
     order = 1
-    def setups(self, to):
+    def setups(self, to:Tabular):
```

Comment:
What's the purpose of the typing here? Do we need typedispatch for these classes?

Reply:
They're not, we don't need type dispatching but I felt like it helps people understand that tabular's get thrown in there (I can of course remove them though!)